calibre (4.0.0+really3.48.dfsg-1) unstable; urgency=medium

  Debian is currently purging all Python2 packages from its archive, with the
  consequence that Python2 packages cannot work properly anymore.
  
  Updating Calibre to newer version is from now on blocked due to
  requirements on Python2 packages that have been removed.

  We recommend users to use upstream Calibre packages to obtain new
  features and security fixes, until upstream together with plugin
  authors switch to Python3. At this point Calibre will be again
  be updated and useable in Debian. 

 -- Norbert Preining <norbert@preining.info>  Sun, 06 Oct 2019 09:31:44 +0900
